app/assets/stylesheets/cart.css.scss

```diff
@@ -88,3 +88,23 @@
   a:visited {color: #008000; text-decoration: underline; }
 }
 
+input[type="submit"].cart-item-quantity-button {
+  border : solid 0px #ffffff;
+  border-radius : .95em;
+  moz-border-radius : 3px;
+  -webkit-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  -moz-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+  font-size : 1em;
+  color : #ffffff;
+  padding : .4em 0;
+  width: 45px;
+  background : #6c64db;
+  background : -webkit-gradient(linear, left top, left bottom, color-stop(0%,#6c64db), color-stop(0%,#5b54b8), color-stop(0%,#544e9c), color-stop(0%,#47428a), color-stop(100%,#544e9c));
+  background : -moz-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+  background : -webkit-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+  background : -o-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
```

Comment:
Color literals like `#6c64db` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#5b54b8` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#544e9c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#47428a` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property name should be immediately followed by a colon<br>Avoid vendor prefixes.